### PR TITLE
Add page Release Notes, and also modify playbook for easier development

### PR DESCRIPTION
**Pull request by gmarpons**

### CODE REVIEW

**Comment by gmarpons** [comment]
It seems to me that PR previews also work better with the proposed change in the playbook.

File: en/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -95,3 +95,5 @@
 
 * xref:en:whitepaper:index.adoc[Whitepaper]
 ** xref:en:whitepaper:decidim-a-brief-overview.adoc[Decidim: a brief overview]
+
+* xref:en:releases:index.adoc[_Release Notes_]
```
Comment (andreslucena) [inline en/modules/ROOT/nav.adoc:99]: Any reason for doing this with italic?

Comment (gmarpons) [reply en/modules/ROOT/nav.adoc:99]: Not really. I'd like a visual hint to differentiate the "Guide" part of the docu and the "Reference" part (e.g. Release Notes, FAQ, Glossary, etc, that I'd keep at the end of the menu). Italics was the simplest solution, but not necessary or an optimal solution.

I can remove it.

Comment (andreslucena) [reply en/modules/ROOT/nav.adoc]: Ah ok, at the moment as this is only "reference" section we'll keep it without italic, in the future when we have other sections we could add it.

**Pull request merged**